training/index.html

```diff
@@ -0,0 +1,75 @@
+---
+layout: default
+title: OMERO Training
+---     
+        {% include navbar-sub-omero.html %}
+        
+        <div class="callout large primary" id="bg-image-omero">
+            <div class="row column text-center">
+                <h1>OME Training</h1>
+                <p>
+                    OME wants to promote understanding of our products with practical
```

Comment:
I will rephrase it and replace ``OME wants`` by something like ``To promote understanding of our products, regular practical  workshops are organised.``